scripts/social-proof.md

```diff
@@ -0,0 +1,35 @@
+# Social Proof
+
+## Hypothesis
+
+Social proof followed by a social commitment (i.e. meet up together, someone else will go if you go) will get people further in the pipeline, hopefully to meet.
+
+Action is collective.
+
+## Metric
+
+More users get past the issue list and further on in the pipeline.
+
+## Script
+
+- Welcome to Movement! I'm Alex from the team, here to get you started. We are currently matching 45 users to events, campaigns and groups in Hackney.
+- `Wait for a response in case they are super keen`
+- In a sec we will send you of list of issues in Hackney. Then, if you want, we will put you in touch with locals who have the same concerns. This is how all change starts - people wanting to change something - together.
+- Are you up for it? If not let us know why and we can try something else to get you involved. 🙂
+
+### Issues List
+
+What are the issues most important to you in Hackney? Pick as many as you want!
+1. 🏥 Healthcare service cuts
+2. 🎓 Education cuts
+3. 💵 Poor wages + working conditions
+4. 👣 Homelessness
+5. 🏚 Rent + housing
+6. ☀ Climate change + pollution
+7. ✋🏾 Racism + migrant rights
+8. ⚧  Gender + sexual discrimination
+9. Something else – tell us!
+
+### After User Pick
+
+- 
```

Comment:
This bit's missing, but we did produce messages for the one-two users who reached past the issue selection. Quickly C&P those lines into here?